src/index.html

```diff
@@ -69,6 +71,10 @@ <h2 class="sidebar-header-name" id="sidebarName">Name Name</h2>
       </section>
       <section class="main-column-activity">
         <section class="infoContainer-cardContainer-card-horizontal">
+          <label for="todays-steps"><input type="text" class="new-activity-input-steps" placeholder="Number of steps" onkeyup="value=value.replace(/[^\d]/g,'')">
```

Comment:
This is really cool! I definitely want you to teach me about this!

Reply:
The label is necessary for our lighthouse score.

Reply:
Great, thank you for putting in those labels